demo/znkQuestionReport/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>znk Question Report</title>
+    <!-- bower:css -->
+    <link rel="stylesheet" href="Swiper/dist/css/swiper.css" />
+    <link rel="stylesheet" href="angular-material/angular-material.css" />
+    <!-- endbower-->
+
+    <link rel="stylesheet" href="main.css">
+
+</head>
+<body ng-app="demo" layout="column"  ng-controller="Main as vm">
+<znk-question-report report-data="vm.questionData" style="margin: 5em;"></znk-question-report>
```

Comment:
I also never use inline style in the real component,
I use it because it's a demo and it's just one line.
However I've move it to style tags